tools/docker/README.md

```diff
@@ -22,4 +22,18 @@ docker build \
   -t vectorized/redpanda-toolchain \
   -f tools/docker/Dockerfile \
   .
+  
+docker run \
+  --rm -ti --privileged \
+  --ipc=host \
+  -v $PWD:$PWD:Z \
+  -w $PWD vectorized/redpanda-toolchain \
+  ./build.sh
+```
```

Comment:
sorry for the confusion. i was thinking having this block before the `If you need to rebuild the toolchain...` (line 18), right after the `... to skip applying SELinux profiles.`